src/IdeaStatiCa.Plugin/BIMPluginHosting.cs

```diff
@@ -211,6 +269,8 @@ private Process RunIdeaIdeaStatiCa(string exePath, string id)
 
 		protected virtual void Dispose(bool disposing)
 		{
+
+			ideaLogger.LogDebug($"BIMPluginHosting Dispose('{disposing}')");
```

Comment:
Move this inside of the `if (!disposedValue)` condition.

Reply:
done